hoodie-client/src/main/java/com/uber/hoodie/table/HoodieMergeOnReadTable.java

```diff
@@ -394,7 +394,7 @@ protected HoodieRollingStatMetadata getRollingStats() {
         } else {
           // If we can index log files, we can add more inserts to log files.
           List<FileSlice> allFileSlices = getRTFileSystemView()
-              .getLatestFileSlicesBeforeOrOn(partitionPath, latestCommitTime.getTimestamp())
+              .getLatestFileSlicesBeforeOrOn(partitionPath, latestCommitTime.getTimestamp(), false)
```

Comment:
With indexable log files, it is safe to route inserts to fileIds in compaction. This should be true. Will change it